scripts/setup.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/sh -ex
+export PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+export DEBIAN_FRONTEND=noninteractive
+export ETCKEEPER_NO_PUSH=1
+
+die() {
+    echo "$@" >&2
```

Comment:
Add a comment saying that the script is terminating and that installation/setup was started but has not completed? On a bare metal machine, dying normally without stating that installation has not completed could lead to things being left over from an old attempt.